### PR TITLE
Implemented API feature

### PR DESCRIPTION
**Pull request by KingsleyIbe**
In this milestone, I implemented the following:

- Created a new game.
- Implement the "Refresh" button.
- Implement the form "Submit" button (adding new data to API).
- Used async and await JavaScript features to consume the API.
- Didn't implement any styling.

### CODE REVIEW

File: src/fetchApi.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const gameId = 'Zl4d7IVkemOTTVg2fUdz';
+
+const postUser = (user, score) => {
```
Comment (GzimAsani) [inline src/fetchApi.js:4]: - [ ] After I add a user and my score, the name does not get added in the leaderboard
- [ ] After I add the user for the first time, I cannot do the same for the second time the information does not get posted to the API.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @KingsleyIbe                               
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hello Gzim,

Thank you for reviewing my project.
The requested changes have been implemented.

**Review by Rustamxon7 — APPROVED** [review]
## Hi @KingsleyIbe   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by KingsleyIbe** [comment]
Hi Rustamjon,

Thank you for approving my project.

**Pull request merged**